pontoon/base/migrations/0040_locale_translators.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import models, migrations
+
+
+def create_translators(apps, schema_editor):
+    """
+    Bug 952488 - We're assigning translation permissions to active contributors.
+    """   
+    User = apps.get_model('auth', 'User')
+
+    # We're using through migration because it's easier to map
+    # user/group primary keys without requiring to query full objects.
+    UserGroup = User.groups.through
+
+    contributors_locale = (
+        User.objects.filter(translation__approved=True)
+            .values('pk', 'translation__locale')
+            .annotate(approved_translations=models.Count('translation__locale', distinct=True))
+            .filter(approved_translations=1)
+    )
+
+    for contributor in contributors_locale:
+        UserGroup.objects.create(user_id=contributor['pk'],
+            group_id=contributor['translation__locale'].group.pk)
```

Comment:
Does this really work? The `values` call should have `['translation__locale']` as an integer with the pk on the locale rather than the locale instance itself.